react/features/base/conference/reducer.js

```diff
@@ -217,36 +214,22 @@ function _conferenceLeftOrWillLeave(state, { conference, type }) {
     // due clean-up like leaving the associated room, but the instance is no
     // longer the focus of the attention of the user and, consequently, the app.
     for (const p in state) {
-        if (p !== 'leaving' && state[p] === conference) {
-            nextState = set(nextState, p, undefined);
+        if (state[p] === conference) {
+            nextState[p] = undefined;
 
             switch (p) {
             case 'conference':
-                // XXX Clear/unset locked & password for a conference which has
-                // been LOCKED_LOCALLY.
-                delete nextState.locked;
-                delete nextState.password;
-                break;
-
             case 'passwordRequired':
                 // XXX Clear/unset locked & password for a conference which has
-                // been LOCKED_REMOTELY.
-                delete nextState.locked;
-                delete nextState.password;
+                // been LOCKED_LOCALLY or LOCKED_REMOTELY.
+                nextState.locked = undefined;
+                nextState.password = undefined;
```

Comment:
The base/redux function `set` will remove properties with value `undefined`, that's why I used delete.

Reply:
I see, makes sense, I'll bring that back in a jiffy.